molecule/system_settings_logging/converge.yml

```
@@ -0,0 +1,28 @@
+---
+- name: converge
+  hosts: all
+  become: true
+  tasks:
+    - name: Converge - Set preserve logs
+      puzzle.opnsense.system_settings_logging:
+        preserve_logs: 10
+
+    - name: Converge - Set max log file size
+      puzzle.opnsense.system_settings_general:
+        max_log_file_size_mb: 15
+
+    # we have no alternative way to compare the values
+    # other than getting theme from the config
```

it seems that you meant "them" instead of "them".
```suggestion
    # other than getting them from the config
```